src/Lean/Elab/Frontend.lean

```diff
@@ -81,23 +81,17 @@ end Frontend
 
 open Frontend
```

Comment:
`processCommand` and the whole `FrontendM` are not used by Lean itself anymore now but I think it's fine to leave them around as they have some external uses. They just will not benefit from parallelism etc. going forward.